algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -448,6 +449,48 @@ case class CMSItem[K: Ordering](item: K, override val params: CMSParams[K]) exte
 }
 
 /**
+ * Used for holding a small set of elements.
+ */
+case class CMSItems[K: Ordering](items: List[K], override val params: CMSParams[K]) extends CMS[K](params) {
+
+  override val totalCount: Long = items.size
```

Comment:
Items is not a sparse counting vector (although that might make sense), but simply a list of keys, which may contain duplicates. So `items.size` is the correct way to get the totalCount - this will be equivalent to the L1-norm of the counting vector once it's transformed into that.